tests/base/utilities_pack_unpack_07.with_zlib=on.output

```diff
@@ -0,0 +1,10 @@
+
+DEAL::unpacked array: 80000
+DEAL::packed array without compression: 80048
```

Comment:
I would suggest to also provide an output variant `utilities_pack_unpack_07.with_zlib=off.output`, which is in principal a copy of this one, but is lacking the lines related to compression.